ospfd/ospf_vty.c

```diff
@@ -171,6 +171,18 @@ static struct ospf *ospf_cmd_lookup_ospf(struct vty *vty,
 	return ospf;
 }
 
+static void ospf_show_vrf_name(struct ospf *ospf, struct vty *vty,
+			       json_object *json)
+{
+	if (ospf->name) {
+		if (json)
+			json_object_string_add(json, "vrfName", ospf->name);
+		else
+			vty_out(vty, "OSPF vrfName: %s\n", ospf->name);
```

Comment:
nit: I think vrfName is ok for JSON output, but for regular CLI output I think something like "VRF Name" would look better. We for example print ospfInstance or OSPF Instance depending if we're using JSON output or not.

Reply:
Thanks Renato. I have changed to VRF Name. 
+                       vty_out(vty, "VRF Name: %s\n", ospf->name);